data/android/local.properties

```diff
@@ -1,2 +1,2 @@
-sdk.dir=/Users/nguyen.huu.nam/Library/Android/sdk
-flutter.sdk=/Users/nguyen.huu.nam/Desktop/Development/SDKs/flutter
\ No newline at end of file
+sdk.dir=/Users/tran.dinh.truyen/Library/Android/sdk
```

Comment:
add `local.properties, Generated.xcconfig, flutter_export_environment.sh`, delete all `.freezed, .g` files and add to git ignored.